helm-charts/common/embedding-usvc/values.yaml

```diff
@@ -5,18 +5,29 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-tei:
-  enabled: false
+# Configurations for OPEA microservice mm-embedding
+# Set it as a non-null string, such as true, if you want to enable logging facility.
+LOGFLAG: ""
 
-replicaCount: 1
+# embedding need to use different backend embedding engine: TEI, multimodal-bridgetower, PredictionGuard
+# Default is to use the TEI(text-embedding-inference) as the backend
+EMBEDDING_BACKEND: "TEI"
 
-# Set it as a non-null string, such as true, if you want to enable logging facility,
-# otherwise, keep it as "" to disable it.
-LOGFLAG: ""
+# Uncomment and set the following settings to use PredictionGuard as the backend
+# EMBEDDING_BACKEND: "PredictionGuard"
+PG_EMBEDDING_MODEL_NAME: "bridgetower-large-itm-mlm-itc"
```

Comment:
Line 18 and 19 shall be commented under the case of 'TEI'?

Reply:
These 2 settings will only be populated into the service if the `EMBEDDING_BACKEND` is "PredictionGuard". I just leave it as the default settings.